onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -791,6 +791,9 @@ extern "C" {
     MLAS_GEMV_U8S8_KERNEL MlasGemvU8S8KernelAvx512Vnni;
     MLAS_GEMM_U8S8_KERNEL MlasGemmU8S8KernelAvxVnni;
     MLAS_GEMV_U8S8_KERNEL MlasGemvU8S8KernelAvxVnni;
+    MLAS_GEMM_U8S8_KERNEL MlasGemmU8U8KernelAvx2Vnni;
```

Comment:
>MlasGemmU8U8KernelAvx2Vnni [](http://example.com/codeflow?start=26&length=26)

nit: could you please rename the above function MlasGemvU8S8KernelAvxVnni and MlasGemmU8S8KernelAvxVnni to Avx2